lib_tflite_micro/src/thread_call.S

```diff
@@ -613,7 +613,9 @@ thread_client_loop:
     ldc     r10, 0
     ssync
     bf      r10, do_not_run
+    {add r4, r1, 0; add r5, r2, 0}
```

Comment:
What does two changes do?

Reply:
Henk added this: it's supposed to allow us not to do thread_setup twice if the arguments don't change

Reply:
Okay, we are backing up the args and restoring them. Alright, thanks!